source/renderer/app/api/api.js

```diff
@@ -1194,6 +1207,10 @@ const _createWalletFromServerData = action(
       balance.total.unit === 'lovelace'
         ? new BigNumber(balance.total.quantity).dividedBy(LOVELACES_PER_ADA)
         : new BigNumber(balance.total.quantity);
+    const walletAvailableAmount =
+      balance.available.unit === 'lovelace'
```

Comment:
Also string `lovelace` => we don't need to define constant for this string literal such as `LOVELACE_LABEL` or whatever acceptable.

Reply:
@yakovkaravelov I added `WalletUnit` declaration and changed flat strings for "lovelace" and "ada" on all places in code. Please take a look